parastell/source_mesh.py

```diff
@@ -375,23 +413,38 @@ def _create_tets_from_hex(self, s_idx, theta_idx, phi_idx):
         # first, ordered clockwise relative to the thumb, followed by the
         # remaining vertex at the end of the thumb.
         # See Moreno, Bader, Wilson 2024 for hexahedron splitting
-        hex_canon_ids = [
-            [idx_list[0], idx_list[2], idx_list[1], idx_list[5]],
-            [idx_list[0], idx_list[3], idx_list[2], idx_list[7]],
-            [idx_list[0], idx_list[7], idx_list[5], idx_list[4]],
-            [idx_list[7], idx_list[2], idx_list[5], idx_list[6]],
-            [idx_list[0], idx_list[2], idx_list[5], idx_list[7]],
+        canonical_ordering_schemes = [
+            [
+                [idx_list[0], idx_list[3], idx_list[1], idx_list[4]],
+                [idx_list[1], idx_list[3], idx_list[2], idx_list[6]],
+                [idx_list[1], idx_list[4], idx_list[6], idx_list[5]],
+                [idx_list[3], idx_list[6], idx_list[4], idx_list[7]],
+                [idx_list[1], idx_list[3], idx_list[6], idx_list[4]],
+            ],
+            [
+                [idx_list[0], idx_list[2], idx_list[1], idx_list[5]],
+                [idx_list[0], idx_list[3], idx_list[2], idx_list[7]],
+                [idx_list[0], idx_list[7], idx_list[5], idx_list[4]],
+                [idx_list[7], idx_list[2], idx_list[5], idx_list[6]],
+                [idx_list[0], idx_list[2], idx_list[5], idx_list[7]],
+            ],
         ]
```

Comment:
Not sure if this is what you had in mind for indexing the canonical ordering scheme @gonuke?